### ISSUE TITLE
How do we learn HourglassDecoder?

### ISSUE THREAD

**Issue by yutanpo1227** [YvanYin/Metric3D#184]
I want to retrain the ConvNext model, but the training script that is given only works with RAFTDecoder.
How to retrain the ConvNext model?

**Comment by zero0kiriyu** [comment]
I have the same question. Wonder how to predict normal map in HourglassDecoder and which loss function is used for its supervision

**Comment by YvanYin** [comment]
In our V1 (hourglass decoder), we did not enclose normal prediction tasks. If you need that, you can directly add a head to predict the normal, and follow ViT-models losses for training.
